mathics/builtin/__init__.py

```diff
@@ -37,6 +38,42 @@
 )
 
 
+def contributing_builtin_var(module, name) -> Optional[type]:
+    """
+    Returns getattr(module, name) if it is a Builtin that
+    should contribute with a definition.
+    Otherwise, returns None.
+    """
+    if name.startswith("_"):
+        return None
+
+    var = getattr(module, name)
```

Comment:
`var` is a pretty generic name. A less generic name would be something like `module_object`.

Reply:
This was the name that this "variable" had when it was "inline" in the callers. Maybe a more suitable name would be `builtin_candidate`